templates/cloudflare-pages/package.json

```
@@ -6,7 +6,7 @@
     "dev:remix": "remix watch",
     "dev:wrangler": "cross-env NODE_ENV=development wrangler pages dev ./public",
     "dev": "remix build && run-p \"dev:*\"",
-    "start": "cross-env NODE_ENV=production npm run dev:wrangler",
+    "start": "cross-env NODE_ENV=production wrangler pages dev ./public",
```

What about having a `wrangler` script that's run in both `dev:wrangler` & `start`?

```json
    "wrangler": "wrangler pages dev ./public"
```

```suggestion
    "start": "cross-env NODE_ENV=production && npm run wrangler",
```

---

@MichaelDeBoey thanks. fixed with https://github.com/remix-run/remix/pull/3537/commits/86b47ebff9ecbc2f107d99ab958cad67c8a5f127

---

Can we put them alphabetically again please?

---

fixed! https://github.com/remix-run/remix/pull/3537/commits/9adc728f2d139dcb5873956002575750c183efea